modules/polaris/src/main/ts/ephox/polaris/pattern/Safe.ts

```diff
@@ -1,9 +1,10 @@
 import { PRegExp } from './Types';
 import * as Unsafe from './Unsafe';
+import { Regex } from '@ephox/katamari';
```

Comment:
This import is out of order and is failing linting rules. If you don't already have it setup, make sure to install the ESLint plugin for your IDE.

Reply:
updated